nipype/pipeline/engine/test_newengine/test_single_node.py

```diff
@@ -0,0 +1,103 @@
+from supernode import Node # or whatever
+
+import numpy as np
+
+import pytest, pdb
+
+def fun1(a, **dict):
```

Comment:
I was calling function like this `self.Interface(**self.inputs)` and was assuming that self.inputs can have other keys, but not sure if we will need it, depends on the final implementation, e.g. not sure where we will store "N1.a" variable.